<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="2" failed="0" total="3" passed="1">
  <reporter-output>
    <line>
      <![CDATA[<br>Verification failure : expected [abc] but found [xyz]<br>]]>
    </line>
    <line>
      <![CDATA[<a target="_blank" href=Wed_May_15_15_57_57_IST_2019.jpg><img src=Wed_May_15_15_57_57_IST_2019.jpg height=200 width=200></img></a>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
    <line>
      <![CDATA[<br>]]>
    </line>
  </reporter-output>
  <suite name="Data Driven Basics" duration-ms="4411" started-at="2019-05-15T10:27:57Z" finished-at="2019-05-15T10:28:01Z">
    <groups>
    </groups>
    <test name="Bank Manager Login Test" duration-ms="4259" started-at="2019-05-15T10:27:57Z" finished-at="2019-05-15T10:28:01Z">
      <class name="com.w2a.testcases.BankManagerLoginTest">
        <test-method status="PASS" signature="bankManagerLoginTest()[pri:0, instance:com.w2a.testcases.BankManagerLoginTest@1c7b90e]" name="bankManagerLoginTest" duration-ms="3753" started-at="2019-05-15T15:57:57Z" finished-at="2019-05-15T15:58:01Z">
          <reporter-output>
            <line>
              <![CDATA[<br>Verification failure : expected [abc] but found [xyz]<br>]]>
            </line>
            <line>
              <![CDATA[<a target="_blank" href=Wed_May_15_15_57_57_IST_2019.jpg><img src=Wed_May_15_15_57_57_IST_2019.jpg height=200 width=200></img></a>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
            <line>
              <![CDATA[<br>]]>
            </line>
          </reporter-output>
        </test-method> <!-- bankManagerLoginTest -->
      </class> <!-- com.w2a.testcases.BankManagerLoginTest -->
    </test> <!-- Bank Manager Login Test -->
    <test name="Add Customer Test" duration-ms="67" started-at="2019-05-15T10:28:01Z" finished-at="2019-05-15T10:28:01Z">
      <class name="com.w2a.testcases.AddCustomerTest">
        <test-method status="SKIP" signature="addCustomerTest(java.util.Hashtable)[pri:0, instance:com.w2a.testcases.AddCustomerTest@162c41e]" name="addCustomerTest" duration-ms="0" started-at="2019-05-15T15:58:01Z" data-provider="dp" finished-at="2019-05-15T15:58:01Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Can't invoke method public java.lang.Object[][] com.w2a.utilities.TestUtil.getData(java.lang.reflect.Method) on this instance of class com.w2a.testcases.AddCustomerTest due to class mismatch]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Can't invoke method public java.lang.Object[][] com.w2a.utilities.TestUtil.getData(java.lang.reflect.Method) on this instance of class com.w2a.testcases.AddCustomerTest due to class mismatch
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:75)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- addCustomerTest -->
      </class> <!-- com.w2a.testcases.AddCustomerTest -->
    </test> <!-- Add Customer Test -->
    <test name="Open Account Test" duration-ms="62" started-at="2019-05-15T10:28:01Z" finished-at="2019-05-15T10:28:01Z">
      <class name="com.w2a.testcases.OpenAccountTest">
        <test-method status="PASS" signature="setUp()[pri:0, instance:com.w2a.testcases.OpenAccountTest@1736069]" name="setUp" is-config="true" duration-ms="18137" started-at="2019-05-15T15:57:39Z" finished-at="2019-05-15T15:57:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="SKIP" signature="openAccountTest(java.util.Hashtable)[pri:0, instance:com.w2a.testcases.OpenAccountTest@1736069]" name="openAccountTest" duration-ms="0" started-at="2019-05-15T15:58:01Z" data-provider="dp" finished-at="2019-05-15T15:58:01Z">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[Can't invoke method public java.lang.Object[][] com.w2a.utilities.TestUtil.getData(java.lang.reflect.Method) on this instance of class com.w2a.testcases.OpenAccountTest due to class mismatch]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: Can't invoke method public java.lang.Object[][] com.w2a.utilities.TestUtil.getData(java.lang.reflect.Method) on this instance of class com.w2a.testcases.OpenAccountTest due to class mismatch
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:75)
	at org.testng.internal.MethodInvocationHelper.invokeDataProvider(MethodInvocationHelper.java:137)
	at org.testng.internal.Parameters.handleParameters(Parameters.java:430)
	at org.testng.internal.Invoker.handleParameters(Invoker.java:1276)
	at org.testng.internal.Invoker.createParameters(Invoker.java:991)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1081)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- openAccountTest -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:com.w2a.testcases.OpenAccountTest@1736069]" name="tearDown" is-config="true" duration-ms="1446" started-at="2019-05-15T15:58:01Z" finished-at="2019-05-15T15:58:03Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- com.w2a.testcases.OpenAccountTest -->
    </test> <!-- Open Account Test -->
  </suite> <!-- Data Driven Basics -->
</testng-results>
